src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * This class encapsulates the Deadline Task.
+ *
+ * @author Megan Wee Rui En
+ * @version CS2103T AY21/22 Semester 1
+ */
+
+package duke.tasks;
+
+import java.time.LocalDate;
+
+public class Deadline extends Task {
+    private final LocalDate dateBy;
```

Comment:
While I don't think this explicitly violates the coding standard, I think it would be clearer to use a variable name such as `dueDate`.